tests/app/main/views/test_index.py

```diff
@@ -35,7 +43,28 @@ def it_should_show_future_events_in_cards(self, client, sample_future_events, sa
 
         assert content == intro_course['title']
 
-    @pytest.mark.parametrize('div_class', ['.navbar', '.footnav'])
+    def it_should_display_text_for_main_article(self, mocker, client, sample_future_events, sample_articles_summary):
+        mocker.patch('app.main.views.index.randint', return_value=0)
+        response = client.get(url_for(
+            'main.index'
+        ))
+        page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+        title = page.select_one("#main_article h2").text
+        content = page.select_one("#main_article p").text
+
+        assert title == sample_articles_summary[0]['title']
+        assert content == sample_articles_summary[0]['short_content'] + " READ MORE"
+
+    def it_should_show_featured_articles_in_cards(self, client, sample_future_events, sample_articles_summary):
+        response = client.get(url_for(
+            'main.index'
+        ))
+        page = BeautifulSoup(response.data.decode('utf-8'), 'html.parser')
+        article_title = page.find("h4", {"id": "article_1"}).string
+        article = [a for a in sample_articles_summary if a['title'] == 'Article title 1'][0]
+        assert article_title == article['title']
```

Comment:
Not sure if you need to search for the article since you are looking at the first article you can probably assume that its going to be the first element - `assert article_title == sample_articles_summary [0]['title']` is clearer and should work. 

This will probably need to change in the future when they can select articles to show but for now it's good enough.